pkg/types/apis/types.go

```diff
@@ -78,3 +78,11 @@ type UpgradeInput struct {
 	PackageName   string `json:"package-name,omitempty"`
 	PackagePath   string `json:"package-path,omitempty"`
 }
+
+type Addon struct {
+	Name        string   `json:"name" wrangler:"required,noupdate"`
+	Description string   `json:"description,omitempty"`
+	FromFile    string   `json:"from-file,omitempty"`
```

Comment:
Do we really need `FromFile` field in API?

Reply:
Done